### PR TITLE
(PDB-3067) Improve command logging

### PR DESCRIPTION
**Pull request by aperiodic**
(**NB:** these commits depend on #2077 and should be merged after it.)

Change the message identifier in the command namespace's log/info
invocations to be a concatenation of the stockpile ID and the timestamp
the message was received at, in order to guarantee uniqueness across
restarts (as the stockpile ID may reset).

Change the overall structure of the logged message in the same form so
it's clear *what* is being logged.

When a command is placed on the queue, log its ID, command name, and
certname at the debug level, similarly to the logging after the command is
processed.

### CODE REVIEW

**Comment by senior** [comment]
@aperiodic I think this needs a rebase, I'm seeing your commits from the other PR

**Comment by aperiodic** [comment]
@senior rebased

File: src/puppetlabs/puppetdb/command.clj [hunk 1]
```diff
@@ -210,8 +216,8 @@
     (jdbc/with-transacted-connection db
       (scf-storage/maybe-activate-node! certname producer-timestamp)
       (scf-storage/add-report! report received))
-    (log/infof "[%s] [%s] puppet v%s - %s"
-               id
+    (log/infof "[%s-%d] '%s' puppet v%s command processed for %s"
+               id, (tcoerce/to-long received)
```
Comment (wkalt) [inline src/puppetlabs/puppetdb/command.clj]: I'm +1 generally, but I don't like this comma :)

Comment (senior) [inline src/puppetlabs/puppetdb/command.clj]: Agreed on the comma. Also, these strings need to be wrapped in a trs call.

Comment (aperiodic) [inline src/puppetlabs/puppetdb/command.clj]: I typically use a comma when some lines have only one argument and other lines have multiple, but I'm not too attached to it. I'll add the `trs`

**Comment by aperiodic** [comment]
Two out of the three jenkins targets passed; in the third, the EC2 instance failed to provision correctly (SSH attempts all timed out). So let's just call it good.

File: src/puppetlabs/puppetdb/command.clj [hunk 2]
```diff
@@ -122,9 +122,12 @@
...
     (.acquire write-semaphore)
     (time! (get @metrics :message-persistence-time)
-           (async/>!! command-chan
-                      (queue/store-command
-                        q command version certname command-stream command-callback)))
+           (let [cmd (queue/store-command
+                       q command version certname command-stream command-callback)
+                 {:keys [id received]} cmd]
+             (async/>!! command-chan cmd)
+             (log/debugf "[%s-%d] '%s' command enqueued for %s"
```
Comment (senior) [inline src/puppetlabs/puppetdb/command.clj]: Missed one trs call here

Comment (senior) [inline src/puppetlabs/puppetdb/command.clj]: Missed one trs call here

**Comment by aperiodic** [comment]
@senior added it

**Pull request merged**